alws/crud/errata.py

```diff
@@ -1034,15 +1025,18 @@ def get_nevra(
                 for repo in pulp_packages[albs_package_pulp_href]:
                     repo_mapping[repo].append(albs_package)
                 albs_package.status = ErrataPackageStatus.released
-                albs_package_nevra = get_nevra(albs_package)
+                albs_package_nevra = get_nevra(
+                    albs_package,
+                    arch=albs_package.errata_package.arch,
+                )
                 albs_records_to_add.add(albs_package_nevra)
         errata_packages.add(errata_package_nevra)
 
     missing_packages = errata_packages.difference(albs_records_to_add)
     if missing_packages:
         msg = (
-            "Cannot release updateinfo record, "
-            "following packages is missing in platform repositories: "
+            "Cannot release updateinfo record, following packages "
```

Comment:
I'd say the message should be: "Cannot release updateinfo record, **the** following packages **are** missing **from** platform"